backend/src/main/java/feedzupzup/backend/notification/event/NotificationEventListener.java

```diff
@@ -0,0 +1,30 @@
+package feedzupzup.backend.notification.event;
+
+import feedzupzup.backend.notification.application.PushNotifier;
+import feedzupzup.backend.notification.domain.NotificationPayload;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class NotificationEventListener {
+
+    private final PushNotifier pushNotifier;
+
+    @EventListener
+    public void handleNotificationEvent(NotificationEvent event) {
```

Comment:
p1)
해당 EventListener에 비동기 어노테이션을 포함하지 않아서, 동기로 동작할 것 같습니다.
그래서 publisher가 이 메서드가 종료될 때까지 대기하는 것으로 예상됩니다!

```suggestion
    @EventListener
    @Async
    public void handleNotificationEvent(NotificationEvent event) {
```
추가해주면 좋을 것 같습니다!